tests/test-class_info.php

```diff
@@ -43,7 +43,7 @@ function test_Import_Status()
 
 	function test_postCountByImportStatus($assert = true)
 	{
-		$entry_xml .= '<div xmlns="http://www.w3.org/1999/xhtml" class="entry" id="gc0a52176-c8fc-4376-b889-0b475a6fe70c"><span class="mainheadword"><span lang="nfr"><a href="#gc0a52176-c8fc-4376-b889-0b475a6fe70c">abani</a></span></span><span class="senses"><span class="sharedgrammaticalinfo"><span class="morphosyntaxanalysis"><span class="partofspeech"><span lang="en">n</span></span></span></span><span class="sensecontent"><span class="sense" entryguid="gc0a52176-c8fc-4376-b889-0b475a6fe70c"><span class="definitionorgloss"><span lang="en">government</span></span><span class="examplescontents"><span class="examplescontent"><span class="example"><span lang="nfr">Ala Ghana abani tia sro pan titi.</span></span><span class="translationcontents"><span class="translationcontent"><span class="translation"><span lang="en">This year there is food help from the Government.</span></span></span></span></span></span></span></span></span></div>';
```

Comment:
On this line the change is from `$entry_xml .= ` to `$entry_xml = ` (changed from concatenation operator to assignment operator). This was causing an error in the unit tests.